docs/user_guide/assessment.rst

```diff
@@ -407,12 +407,55 @@ to take advantage of the inherent plotting capabilities of
 
 .. literalinclude:: ../auto_examples/plot_quickstart.py
     :language: python
-    :start-after: # Analyze metrics using MetricFrame
```

Comment:
@riedgar-ms was rather attached to the `:start-after:` tag. Perhaps we can keep that and use `:end-before` for the end? Or is this acceptable to you @riedgar-ms ?
I'm personally fine with  `:lines:`

Reply:
sure not a problem, I switched them back!